packages/components/src/mobile/cycle-select-control/index.native.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import CyclePickerCell from '../bottom-sheet/cycle-picker-cell';
+
+function CycleSelectControl( {
+	help,
+	instanceId,
+	label,
+	multiple = false,
+	onChange,
+	options = [],
+	className,
+	hideLabelFromVision,
+	...props
+} ) {
+	const id = `inspector-select-control-${ instanceId }`;
+
+	return (
+		<CyclePickerCell
```

Comment:
I'm not sure I see the benefit in having both a `CycleSelectControl` and a `CyclePickerCell`, they seem mostly identical.

Reply:
I think I'd prefer to keep it this way for now. It matches other Control / BottomSheet Cell pairs like ToggleControl / SwitchCell and I think there may be implications later on related to cross platform controls, where the control might become independent of where it lives in the UI. This issue has some interesting conversation on that topic https://github.com/WordPress/gutenberg/issues/18018